### ISSUE TITLE
Pedro Tabacof - Unlocking the Power of Gradient-Boosted Trees (using LightGBM) | PyData London 2022

### ISSUE THREAD

**Issue by npatta01** [numfocus/YouTubeVideoTimestamps#115]
**Video Title:**
Pedro Tabacof - Unlocking the Power of Gradient-Boosted Trees (using LightGBM) | PyData London 2022

**Video Link:**
https://www.youtube.com/watch?v=qGsHlvE8KZM&t=2s&ab_channel=PyData

**Contents:**
00:10 Introduction of speaker 
00:15 Speaker begins talk
00:50 Why LightGBM
01:30 Speaker bio
02:20 Agenda
02:40 Example of LightGBM on Kaggle house priices prediction
04:43 Kaggle gradient-boosted trees performance
07:00 How gradient boosted trees work
09:36 What are the important tuning parameters
11:15 Handling missing values
13:41 Handling categorical features
15:35 Supported Loss functions
18:58 Custom loss functions
20:24 Early stopping
20:58 Sample / class weights
22:22 Other cool features in LightGBM
23:35 Interpretability: SHAP
25:06 Conclusion and takeway
27:59 Q&A
